contrib/elftoolchain/libdwarf/libdwarf_init.c

```diff
@@ -302,7 +302,7 @@ _dwarf_alloc(Dwarf_Debug *ret_dbg, int mode, Dwarf_Error *error)
 {
 	Dwarf_Debug dbg;
 
-	if ((dbg = calloc(sizeof(struct _Dwarf_Debug), 1)) == NULL) {
+	if ((dbg = calloc(1, sizeof(struct _Dwarf_Debug))) == NULL) {
```

Comment:
likewise this should go to https://sourceforge.net/projects/elftoolchain/

Reply:
Fixed!